packages/banner/src/Banner.tsx

```diff
@@ -306,9 +40,13 @@ export default function Banner({
   image,
   children,
   className,
+  darkMode: darkModeProp,
+  baseFontSize: baseFontSizeProp,
   ...rest
 }: BannerProps) {
-  const { icon: Icon, color } = map[variant];
+  const { theme, darkMode } = useDarkMode(darkModeProp);
+  const baseFontSize = useUpdatedBaseFontSize(baseFontSizeProp);
+  const { icon: Icon, color } = map[theme][variant];
 
   const renderIcon = image ? (
```

Comment:
This image ends up being full height. Is this intentional? 
<img width="125" alt="Screen Shot 2022-09-02 at 2 18 10 PM" src="https://user-images.githubusercontent.com/2414030/188214318-5b6502cb-f2e2-4b47-a030-98688206b6c6.png">

Reply:
This is the custom image, right? I think it always functioned like this,  plus Rob created a ticket to deprecate this feature so I don't think we have to worry much about this.